Routing/Loader/RestYamlCollectionLoader.php

```diff
@@ -94,9 +96,9 @@ public function load($file, $type = null)
                     $this->collectionParents[$name] = $parents;
                 }
 
-                if ($options) {
```

Comment:
I guess so. `addOptions()` does nothing spectacular, and that method already checks if `$options` is empty by [itself](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Routing/RouteCollection.php#L235) before iterating over its routes.